docs/source/conf.py

```diff
@@ -105,19 +127,43 @@ def copy_html_files(app, exception):
     """
     Copy all .html files from source to build directory, maintaining the directory structure.
     """
-    print("Copying HTML files to build directory")
+    copy_print_info = "Copying HTML files to build directory"
+    print()
+    print(copy_print_info)
+    print(len(copy_print_info) * "=")
```

Comment:
Is this for debug purpose?

Reply:
I would say it is on the info level. I would keep it, because it useful information and makes more transparent what happens to the user. I did not realize that the html files by this are copied until I saw the code. Maybe there is a better way to integrate it with the other sphinx messages, but for now this is okay.

Reply:
Actually there seems to be sphinx logging utils. I make an issue for this.
https://www.sphinx-doc.org/en/master/extdev/logging.html

Reply:
Made an issue #244